readthedocs/projects/exceptions.py

```diff
@@ -8,13 +11,20 @@
 
 class ProjectConfigurationError(BuildEnvironmentError):
 
-    """Error raised trying to configure a project for build"""
+    """Error raised trying to configure a project for build."""
 
     NOT_FOUND = _(
         'A configuration file was not found. '
         'Make sure you have a conf.py file in your repository.'
     )
 
+    MULTIPLE_CONF_FILES = _(
+        'There are more than one conf.py file and none of them say doc '
+        "in their path, we don't know which one use. Please, select "
+        'the correct one under the Advanced settings tab in the '
+        "project's Admin."
```

Comment:
A few notes on copy here:

- More clarity around *why* this is a problem is good -- that is: "We found more than one conf.py and are not sure which one to use." instead of mentioning `docs/`
- "Please, select the correct one" -> "Please specify the correct file"